### PR TITLE
commiting index page of quiz app

### PR DESCRIPTION
**Pull request by rajaranjanmr**

### CODE REVIEW

**Review by sri-rishi — CHANGES_REQUESTED** [review]
image

I think you should work with colors UI is not looking good, and card are not centered

File: index.css [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+    padding:0;
+    margin:0;
+    box-sizing: border-box;
+    letter-spacing: 1px;
+    color:aliceblue;
+}
+body{
+    background-color: #F6830F;
+
+
```
Comment (sri-rishi) [inline index.css:11]: remove all  extra spacing from code

File: index.css [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+    width: 100%;
+    text-align: center;
+    /* margin-top:1rem; */
+}
+.container-quiz h1{ 
+    color: aliceblue;
+    font-size: 4rem;
+    font-weight: 600;
+    text-align: center;
+    /* position: relative; */
```
Comment (sri-rishi) [inline index.css:23]: remove all comments

File: index.css [hunk 3]
```diff
@@ -0,0 +1,327 @@
...
+    gap:1rem;
+
+
+}
+.category a{
+    text-decoration:none;
+    font-size: 2rem;
+    color:white;
+    cursor:pointer;
+    background-color: #612897;
```
Comment (sri-rishi) [inline index.css:77]: use variable color naming

**Review by sri-rishi — CHANGES_REQUESTED** [review]

File: index.js [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+    questionEl.innerText = currentQuizData.question
+    a_text.innerText = currentQuizData.a
+    b_text.innerText = currentQuizData.b
+    c_text.innerText = currentQuizData.c
+    d_text.innerText = currentQuizData.d
+}
+
+function deselectAnswers() {
+    answerEls.forEach(answerEl => answerEl.checked = false)
+}
```
Comment (Anurag021) [inline index.js:68]: there are 2 different variations of defining functions (i.e. arrow(=>), and function) try to follow the same standard,
I guess arrow one would be fine here

**Review by Anurag021 — APPROVED** [review]
I believe specific use of theme in color would look better on UI, and try to make use of variables for color that would bring uniformity to the code, rest looks good.

**Review by sri-rishi — APPROVED** [review]

**Pull request merged**